src/views/Teams/TeamList.jsx

```diff
@@ -0,0 +1,84 @@
+import { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import { deleteTeamById, getTeams } from '../../services/teams';
+
+function TeamList() {
+  const [loading, setLoading] = useState(true);
+  const [teams, setTeams] = useState([]);
+
+  const loadTeams = async () => {
+    setLoading(true);
+    const resp = await getTeams();
+    setTeams(resp);
+    setLoading(false);
+  };
+
+  useEffect(() => {
+    loadTeams();
+  }, []);
+
+  const handleDelete = async ({ id, name }) => {
```

Comment:
When a user tries to delete a team with any players associated, it doesn't work and there isn't a user-facing error that says what happened. You can handle this in a few ways: Add an error that tells the user to delete the players first, add player deletion to the team delete handler function, etc. Up to you